qemu_v8.mk

```diff
@@ -255,6 +280,9 @@ linux-cleaner: linux-cleaner-common
 # OP-TEE
 ################################################################################
 OPTEE_OS_COMMON_FLAGS += DEBUG=$(DEBUG) CFG_ARM_GICV3=$(GICV3)
+ifeq ($(XEN_BOOT),y)
+OPTEE_OS_COMMON_FLAGS += CFG_VIRTUALIZATION=y CFG_CORE_DYN_SHM=y
```

Comment:
Isn't `CFG_CORE_DYN_SHM=y` the default value already?

Reply:
Yes, I will remove it.